mlxtend/evaluate/bootstrap_point632.py

```diff
@@ -153,21 +165,39 @@ def bootstrap_point632_score(estimator, X, y, n_splits=200,
             raise AttributeError('Estimator type undefined.'
                                  'Please provide a scoring_func argument.')
 
+    # determine which prediction function to use
+    # either label, or probability prediction
+    if not predict_proba:
+        predict_func = cloned_est.predict
+    else:
+        if not getattr(cloned_est, 'predict_proba', None):
+            raise RuntimeError(f'The estimator {cloned_est} does not '
+                               f'support predicting probabilities via '
+                               f'`predict_proba` function.')
+        predict_func = cloned_est.predict_proba
+
     oob = BootstrapOutOfBag(n_splits=n_splits, random_seed=random_seed)
     scores = np.empty(dtype=np.float, shape=(n_splits,))
     cnt = 0
     for train, test in oob.split(X):
         cloned_est.fit(X[train], y[train])
 
-        test_acc = scoring_func(y[test], cloned_est.predict(X[test]))
+        # get the prediction probability
+        # for binary class uses the last column
```

Comment:
Doe the current implementation support multi-class settings? If not, I think we need the following structure?

```python
if predict_proba:
    len_uniq = np.unique(y)

    if len(len_uniq) < 2:
        # raise error
    elif len(len_uniq) == 2:
         predicted_train_val = predicted_train_val[:, 1]
         predicted_test_val = predicted_test_val[:, 1]
    else:
        # do something

Reply:
Don't all `sklearn` estimators check the degenerate case of a single-class classification? If multiclass, then we can return the full probability array, else in binary classification, get the 2nd column.

I took the general structure though. lmk what  you think

Reply:
> Don't all sklearn estimators check the degenerate case of a single-class classification? 

I think you are right!

Looking at https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

> the multiclass and multilabel cases expect a shape (n_samples, n_classes). 

it seems to be okay as it is right now, too.